pkg/tracer/tracer_linux.go

```diff
@@ -26,21 +29,47 @@ type LinuxTracer struct {
 	// In flight connections are the connections that already existed before the EBPF module was loaded.
 	// These connections are stored with a key without direction, to make it possible to merge with undirected
 	// metric stats
-	inFlightTCP map[string]*common.ConnectionStats
+	inFlightTCP     map[string]*common.ConnectionStats
+	metricsGatherer prometheus.Gatherer
+
+	perfEventsBytes   chan []byte
```

Comment:
Some comment (like for `inFligthTcp`) to describe what's in this and the next few variables would be super helpful